packages/flutter_tools/templates/create/ios.tmpl/Podfile

```diff
@@ -0,0 +1,10 @@
+# Uncomment this line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'Runner' do
+  # Uncomment this line if you're using Swift or would like to use dynamic frameworks
+  # use_frameworks!
```

Comment:
This comment is in the default Podfile you get if you say "pod install" so I've left it. If we ever turn Flutter into a Pod we'll want to revisit